### PR TITLE
[fix](lookup)enhance compatibility for lookup join with the java.sql.Timestamp 

### PR DESCRIPTION
**Pull request by vinlee19**
# Proposed changes
When you use a lookup join in Flink SQL to read a timestamp from a table in Doris as a dimension table, you may encounter the following error: `timestamp type must be java.time.LocalDate", while the actual type is java.sql.Timestamp.` This is because there is no compatibility  java.sql.Timestamp.
## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by JNSimba** [comment]
Thanks for @vinlee19, can you write  the description of pr

**Review by JNSimba — DISMISSED** [review]
LGTM

File: flink-doris-connector/src/main/java/org/apache/doris/flink/deserialization/converter/DorisRowConverter.java [hunk 1]
```diff
@@ -195,9 +196,11 @@ public DeserializationConverter createInternalConverter(LogicalType type) {
                 return val -> {
                     if (val instanceof LocalDateTime) {
                         return TimestampData.fromLocalDateTime((LocalDateTime) val);
+                    } else if (val instanceof Timestamp) {
+                        return TimestampData.fromTimestamp((Timestamp) val);
                     } else {
                         throw new UnsupportedOperationException(
-                                "timestamp type must be java.time.LocalDateTime, the actual type is: "
+                                "timestamp type must be java.time.LocalDateTime or java.time.Timestamp, the actual type is: "
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/deserialization/converter/DorisRowConverter.java:203]: is java.sql.timestamp ?

Comment (vinlee19) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/deserialization/converter/DorisRowConverter.java]: done

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**